project-fob/Controllers/StatsController.cs

```diff
@@ -23,39 +23,36 @@ public ActionResult Index()
             return View();
         }
         public string GetStats()
-        {
-            { //- for in the same topic, : for different topic, ; for completely different statistic
-
-                byte[] meetingIdValue;
-                bool gotvalue = false;
-                gotvalue = HttpContext.Session.TryGetValue("meetingid", out meetingIdValue);
+        { //- for in the same topic, : for different topic, ; for completely different statistic
 
-                string session = meetingIdValue.ToString();
-                //string session = Session["meetingid"].ToString();
+            bool gotvalue = false;
+            gotvalue = HttpContext.Session.TryGetValue("meetingid", out var meetingIdValue);
```

Comment:
The bool `gotvalue` doesn't seem to be used here, so might as well get rid of it.